aws/resource_aws_iot_topic_rule.go

```diff
@@ -124,11 +124,11 @@ func resourceAwsIotTopicRule() *schema.Resource {
 						},
 						"range_key_field": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Can you please either update any existing acceptance testing to omit these values or preferably add an acceptance test that covers omitting these values successfully? Thanks!

Reply:
Ah good catch!